tests/test_cachito.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import unittest
+
+import cachito
+
+
+class CachitoTestCase(unittest.TestCase):
```

Comment:
I'd rather use pytest tests and pytest fixtures for the test client like we do in Greenwave and Estuary